src/components/Header/index.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import { Link } from "gatsby";
+import logo from '../../../public/images/shared/logo.svg';
+import hamburgerMenu from '../../../public/images/shared/icon-hamburger.svg';
+import iconClose from '../../../public/images/shared/icon-close.svg';
+import Navigation from '../Navigation';
+
+const Header = () => {
+
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggleMenu = () => {
+    setIsOpen(!isOpen);
+  }
+
+  return (
+    
+    <header className="header">
+      <a href="#main" className="skip__links">Skip to main content</a>
+      <Link to="/">
+        <img src={logo} alt="Space travel logo" className="header__logo"></img>
+      </Link>
+      <button className="header__menu" aria-expanded={isOpen ? "true" : "false"} aria-controls="navigation">
+        <span className="sr-only">Menu</span>
+          <img className={isOpen ? "header__menu--img" : "header__menu--img active"} src={hamburgerMenu} alt="hamburger menu" onClick={toggleMenu}></img>
```

Comment:
You have separate buttons for opening/closing, so you should not toggle the state. The open button should always set the state variable to `true` and the close button should always set the variable to `false`. Otherwise, you might get into an inconsistent state where the open button will actually cause the hamburger menu to close.